assets/js/util.js

```diff
@@ -242,8 +243,26 @@ function synchronizeTextareaHeights() {
     $('#translatedText').css('height', originalTextScrollHeight + 'px');
 }
 
-/*:: export {synchronizeTextareaHeights, modeEnabled, ajaxSend, ajaxComplete, filterLangList, onlyUnique,
-    SPACE_KEY_CODE, ENTER_KEY_CODE, HTTP_OK_CODE, HTTP_BAD_REQUEST_CODE, XHR_LOADING, XHR_DONE} */
+function apyCall(requestObject, endpoint) {
+    window[requestObject.requestName] = $.ajax({
```

Comment:
> Would it make sense to pass requestName as an argument to callApy itself?

Yes; I think I mentioned this earlier.

> I could write the complete: function() { } inside the target object itself directly then!

I think we should leave that to the caller.